features/custom-ellipsis.yml

```
@@ -0,0 +1,5 @@
+name: Custom ellipsis
```

tiniest nit: I think it'd be nice to make this (and the identifier) plural? Since you're (presumably) using this to elide text generally, not for individual texts (otherwise, you'd just manually elide the text).

```suggestion
name: Custom ellipses
```